datafusion/src/physical_plan/planner.rs

```diff
@@ -1422,6 +1422,10 @@ impl DefaultPhysicalPlanner {
             new_plan = optimizer.optimize(new_plan, &ctx_state.config)?;
             observer(new_plan.as_ref(), optimizer.as_ref())
         }
+        debug!(
+            "Optimized physical plan short version:\n{}\n",
+            displayable(new_plan.as_ref()).indent().to_string()
```

Comment:
```suggestion
            displayable(new_plan.as_ref()).indent()
```

I don't think the call to `to_string()` is needed here -- not that it is a huge deal